src/components/ERACommons.jsx

```diff
@@ -135,7 +140,7 @@ export default function ERACommons(props) {
           <span style={{verticalAlign: '50%'}}>Authenticate your account</span>
         </a>
       )}
-      {nihError && <span className='era-cancel-color era-required-field-error-span'>{nihErrorMessage}</span>}
+      {nihError && <span data-cy='era-commons-error-span' className='era-cancel-color era-required-field-error-span'>{nihErrorMessage}</span>}
```

Comment:
The only purpose of `nihError` is here, to gate the showing of an error message. It looks like this change removes the message used by the other error cases (error on missing decoded token, error on delete account linkage error). I would fix both of these issues by removing `nihError` and use `nihErrorMessage` instead of `nihError` here, and add new error messages to the two other cases that are currently setting `nihError`.